velero-plugins/common/util.go

```diff
@@ -152,3 +152,15 @@ func GetOwnerReferences(item runtime.Unstructured) ([]metav1.OwnerReference, err
 	}
 	return metadata.GetOwnerReferences(), nil
 }
+
+// ConfigureContainerSleep replaces the pod, cmd, and arg on containers so that
+// post stage or migrate phase 1 restores applications do not start
+func ConfigureContainerSleep(containers []corev1API.Container, duration string) {
```

Comment:
I think we can remove this function now. This shouldn't be used any longer.

Reply:
I updated util.go. Please check and let me know if anything else is required